app/models/option_node.rb

```diff
@@ -245,6 +245,12 @@ def arrange_as_rows(hash = nil, parent_path = [])
     end
   end
 
+  def preferred_name_translations(path)
+    path.map do |path|
+      path.name_translations[configatron.preferred_locales.first.to_s]
```

Comment:
We should really add a `mission_preferred_locale` entry to configatron. Would be much cleaner and reusable.